api/app/src/app.ts

```diff
@@ -1,20 +1,33 @@
 import * as dotenv from "dotenv";
 dotenv.config();
 // Keep dotenv import and config before everything else
+import * as Sentry from "@sentry/node";
 import cors from "cors";
 import express, { Application, Request, Response } from "express";
 import helmet from "helmet";
 import { errorHandler } from "./default-error-handler";
 import initDB from "./models/db";
 import mountRoutes from "./routes/index";
+import { initSentry, isSentryEnabled } from "./sentry";
+import { Config } from "./shared/config";
 
 const app: Application = express();
+const version = Config.getVersion();
+
+// Must be before routes
+initSentry(app);
+
 app.use(helmet()); // needs to come before any route declaration, including cors()
 app.use(express.json({ limit: "2mb" }));
 app.use(express.urlencoded({ extended: false, limit: "2mb" }));
 app.use(cors());
 app.set("etag", false);
 
+app.use((req, res, next) => {
+  version && res.setHeader("X-Metriport-Version", version);
+  next();
+});
```

Comment:
Heads up, with this we should have our version available on our API's responses. 🤩